pyflakes/test/test_other.py

```diff
@@ -1993,3 +1994,18 @@ def test_raise_notimplemented(self):
         self.flakes('''
         raise NotImplemented
         ''', m.RaiseNotImplemented)
+
+
+class TestMaximumRecursion(TestCase):
+
+    def setUp(self):
+        self._recursionlimit = sys.getrecursionlimit()
+
+    def test_recursion_limit(self):
+        # Using self._recursionlimit * 10 tends to cause CPython to core dump.
+        r = range(self._recursionlimit * 9)
```

Comment:
Lots of segmentation fault now after a rebase, on `self._recursionlimit * 9`

I'll try reducing it.